resotocore/resotocore/cli/command.py

```diff
@@ -1983,10 +1989,32 @@ def fmt_json(elem: Json) -> JsonElement:
             else:
                 return elem
 
-        def fmt(elem: JsonElement) -> JsonElement:
-            return fmt_json(elem) if isinstance(elem, dict) else str(elem)
+        async def fmt_csv(elem: Json, idx: int) -> AsyncIterator[JsonElement]:
+            def to_csv_string(lst: List[Any]) -> str:
```

Comment:
Mmh idx is actually not required - you could do something like this:
```python
async def csv_stream(in_stream: Stream) -> AsyncIterator[JsonElement]:
   yield "header"
   async for elem in in_stream:
       yield to_csv(elem)
```
Wouldn't this be simpler?